scripts/ci/workflows-issues-manager/constants.js

```diff
@@ -0,0 +1,17 @@
+const DEFAULT_K = 5;
+
+const MIN_RUNS = {
```

Comment:
I understood this are workflow identifier and the unstable threshold parameter. A more detailed comment string will help to understand.

Reply:
Done 👍